lua/go/config.lua

```diff
@@ -12,6 +12,8 @@ M.options = {
     lint_prompt_style = 'qf',
     -- formatter: goimports, gofmt, gofumpt
     formatter = 'goimports',
+    -- maintain cursor position after formatting loaded buffer
+    maintain_cursor_pos = true,
```

Comment:
I think it better to change default to `false`, because it's new feature somehow breaking.